api/v1alpha1/gatekeeper_types.go

```diff
@@ -214,26 +197,69 @@ type WebhookConfig struct {
 	// +optional
 	DisabledBuiltins []string `json:"disabledBuiltins,omitempty"`
 
+	// Deprecated: spec.webhook.logMutations is deprecated. Use the field at
+	// spec.mutatingWebhookConfig.logMutations, which will override what is set here.
 	// LogMutations enables the logging of mutation events and errors. The default value is Disabled.
 	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#beta-enable-mutation-logging-and-annotations.
 	//
-	// +optional
+	// +kubebuilder:default=Disabled
 	//nolint:lll
-	LogMutations *Mode `json:"logMutations,omitempty"`
+	LogMutations Mode `json:"logMutations,omitempty"`
 
+	// Deprecated: spec.webhook.mutationAnnotations is deprecated. Use the newer field at
```

Comment:
(nit) newer is implied
```suggestion
	// Deprecated: spec.webhook.mutationAnnotations is deprecated. Use the field at
```